Makefile

```diff
@@ -33,6 +37,7 @@ $(PROTODIR):
 clean:
 	$(MAKE) -C $(PROTODIR) clean
 	$(RMRF) node
```

Comment:
@stepanvalyavskiy and we need to update binary name for the back-end too `degitx`